en/sql/schema/index_stmt.rst

```diff
@@ -327,9 +327,9 @@ The following is a syntax of rebuilding an index.
 
 ::
 
-    ALTER INDEX index_name ON [schema_name.]table_name REBUILD;
+    ALTER INDEX index_name ON [schema_name.]table_name {{[COMMENT index_comment_string] REBUILD} | {COMMENT index_comment_string} | VISIBLE | INVISIBLE};
 
-*   *index_name*: specifies the name of the index to be recreated. The index name must be unique in the table.
+*   *index_name*: specifies the name of the index to be altered or recreated. The index name must be unique in the table.
```

Comment:
How about changing "recreated" to "rebuilt" ?

Reply:
OK